README.md

```diff
@@ -21,16 +21,31 @@ These are important benefits:
    source-level diff, LLpatch uses LLVM IR for diffing to abstract away all the
    specifics/details on machine architecture, types of linkers, and kernel version.
 
-2. **Slim Livepatch**: LLpatch uses 'llvm-diff' to perform the diffing and generates LLVM IR
+2. **Easy Callbacks and LLPatch Symbols**: Kernel livepatch allows four callbacks
+   before/after patching/unpatching the livepatch. LLpatch helps livepatch developer to
+   implement those callbacks in their local .c file and feed it to LLpatch. In addition,
+   in their callbacks, LLpatch provides simple macro to access global variables (doesn't
+   matter they are local static or not) defined in kernel module and/or vmlinux in a
+   "running" machine. This helps and simplies development for kernel livepatch.
+
+3. **Slim Livepatch**: LLpatch uses 'llvm-diff' to perform the diffing and generates LLVM IR
    code that distills the differences. This allows compiler-backend to fully optimize a
    final ELF binary for kernel livepatch. (e.g., no redundant global variables not used in
    livepatch.
 
-3. **Better Checking and Validation**: LLVM IR contains much more information than ELF
+4. **Better Checking and Validation**: LLVM IR contains much more information than ELF
    binary. So, it's possible to implement lots of useful checking and validation mechanism
    for livepatch generation. (e.g., changes in function prototype should not be allowed.)
 
-4. **Semi-Automatic Livepatch Generation and Easy Debugging**
+5. **Ultra-Fast Livepatch Generation and Unchanged Kernel Repository**
+   + LLpatch doesn't require special compiler options for the diffing. So, kernel doesn't
+     need to be re-compiled differently.
```

Comment:
s/So, kernel/So the kernel/ and remove the "differently"

Reply:
Done.